_Streams/114.1-neural-networks-continued.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Neural Networks Continued"
+video_number: 114.1
+date: 2018-01-12
+video_id: iCxqLVm2H3w
+
+topics:
+  - title: "Improving Matrix class"
+    time: "21:11"
+    url: "https://www.youtube.com/watch?v=HuZbYEn8AvY"
+  - title: "Feed Forward Neural Networks Pt.1"
```

Comment:
This should be "Part" instead of "Pt." since this convention is used everywhere else.